imod/mf6/api_package.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from imod.mf6.package import Package
+from imod.schemata import (
+    DTypeSchema,
+)
+
+_init_schemata = {
+    "maxbound": [DTypeSchema(np.integer)],
+}
+
+class ApiPackage(Package):
+    """
```

Comment:
The docstring still lacks a general explanation what the package does. I think your message in the changelog is a good start: 

"interact with libMF6.dll through its API."

Could you write that to a full sentence for the general explanation?

Reply:
done